app/models/picture.rb

```diff
@@ -0,0 +1,6 @@
+class Picture < ActiveRecord::Base
+  has_attached_file :image,
+                    :styles => { :medium => "300x300>", :thumb => "100x100>" },
+                    :default_url => "/images/:style/missing.png"
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)